esm/interface/css-style-declaration.js

```diff
@@ -81,6 +81,21 @@ export class CSSStyleDeclaration extends Map {
     refs.get(this).setAttribute('style', value);
   }
 
+  getPropertyValue(name) {
+    const self = this[PRIVATE];
+    return handler.get(self, name);
```

Comment:
The handler is for proxy thing ... once you have the original *Map* that is holding all values, you gotta do like:

```js
return self.get(uhyphen(name));
```

got it?

Reply:
Actually, never mind ... `updateKeys` needs to happen, so this was unnecessary optimization. All good!